spec/system/jobseekers/jobseekers_can_manage_a_profile_spec.rb

```diff
@@ -342,12 +325,10 @@
           visit edit_jobseekers_profile_work_history_path(employment)
         end
 
-        it "shows the old data" do
+        it "successfully changes the employment record" do
```

Comment:
Combined these two tests as I don't think we needed one to just check two lines of content on it's own, and combining them may slightly reduce run time.